content/en/providers/check-in/sp/_index.md

```diff
@@ -926,14 +928,14 @@ the API described at the
 This section defines the attributes that can be made available to services
 connected to Check-in.
 
-### 1. EGI ID
-
 <!-- markdownlint-disable line-length no-inline-html -->
 
+### 1. EGI ID
+
 |          attribute name | EGI ID                                                                              |
 | ----------------------: | :---------------------------------------------------------------------------------- |
 |         **description** | An identifier for the user, unique among all EGI accounts and never reused          |
```

Comment:
**description** | The User’s Community Identifier is a globally unique, opaque, persistent and non-reassignable identifier that identifies the user. For users whose community identity is managed by Check-in, this identifier is of the form `<uniqueID>@egi.eu`. The `<uniqueID>` portion is an opaque identifier issued by Check-in.